test/contract.js

```diff
@@ -21,38 +23,56 @@ describe('Contract', () => {
       assert.equal(contract.abi, ContractMetadata.EventFactory.abi);
     });
 
-    it('removes the hex prefix from the address', async () => {
+    it('removes the hex prefix from the address', () => {
       contract = new Contract(Config.QTUM_RPC_ADDRESS, '0x1234567890', ContractMetadata.EventFactory.abi);
       assert.equal(contract.address, '1234567890');
     });
   });
 
   describe('call()', () => {
-    it('returns the values', async () => {
+    it('returns the values', () => {
+      // Mock return from CentralizedOracle.bettingStartBlock()
+      const result = {
+        "address": "d78f96ea55ad0c8a283b6d759f39cda34a7c5b10",
+        "executionResult": {
+          "gasUsed": 22060,
+          "excepted": "None",
+          "newAddress": "d78f96ea55ad0c8a283b6d759f39cda34a7c5b10",
+          "output": "00000000000000000000000000000000000000000000000000000000000100e0",
+          "codeDeposit": 0,
+          "gasRefunded": 0,
+          "depositSize": 0,
+          "gasForDeposit": 0
+        },
+        "transactionReceipt": {
+          "stateRoot": "81972b5d65f1c12853c0324fbe5e9b2233843431f533a2b27351842e02247e1a",
+          "gasUsed": 22060,
+          "bloom": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+          "log": [
+          ]
+        }
+      };
+
       contract = new Contract(
-        Config.QTUM_RPC_ADDRESS, 'dacd16bde8ff9f7689cb8d3363324c77fbb80950',
-        ContractMetadata.TopicEvent.abi,
+        Config.QTUM_RPC_ADDRESS, 
+        'd78f96ea55ad0c8a283b6d759f39cda34a7c5b10', 
+        ContractMetadata.CentralizedOracle.abi
       );
-
-      const res = await contract.call('getEventName', {
-        methodArgs: [],
-        senderAddress: Config.SENDER_ADDRESS,
-      });
-      assert.equal(res[0].replace(/\0/g, ''), 'Who will win the 2018 NBA Finals Championships?');
+      const formatted = Formatter.callOutput(result, ContractMetadata.CentralizedOracle.abi, 'bettingEndBlock', true);
+      assert.isTrue(Web3Utils.isBN(formatted['0']));
+      assert.equal(formatted['0'].toJSON(), '100e0');
```

Comment:
Ideally this "100e0" should not be hardcoded. The expected output and input should be generated from the same source in test.

For example,
in the beginning of test.
var input = "0x00000000100e0";
var output = convertToString(input);

in the end
assertEqual with output.

Reply:
I've approved this PR. If you decide to make changes, make sure test still passes and merge.